# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xlockmore/no-.plan.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We decided exposing the (maybe) slightly personal/secret .plan is a bad idea.

  - Rene Rebe <rene@exactcode.de>

diff -ur xlockmore-5.14.1/xlock/xlock.c xlockmore-5.14.1-fixed/xlock/xlock.c
--- xlockmore-5.14.1/xlock/xlock.c	2004-11-09 23:07:06.000000000 +0100
+++ xlockmore-5.14.1-fixed/xlock/xlock.c	2005-01-11 16:00:32.543487472 +0100
@@ -2963,6 +2963,8 @@
 	(void) sprintf(buffer, "%s/%s", home, ".xlocktext");
 #endif
 	planf = my_fopen(buffer, "r");
+
+#ifdef	PLAN_WANTED	/* disable .plan exposing for T2 -ReneR */
 	if (planf == NULL) {
 #ifdef VMS
 		(void) sprintf(buffer, "%s%s", home, ".plan");
@@ -2971,6 +2973,7 @@
 #endif
 		planf = my_fopen(buffer, "r");
 	}
+#endif /* PLAN_WANTED */
 	if (planf == NULL) {
 #ifndef VMS
 		(void) sprintf(buffer, "%s/%s", home, ".signature");
diff -ur xlockmore-5.14.1/xlock/xlock.man xlockmore-5.14.1-fixed/xlock/xlock.man
--- xlockmore-5.14.1/xlock/xlock.man	2004-11-02 02:23:21.000000000 +0100
+++ xlockmore-5.14.1-fixed/xlock/xlock.man	2005-01-11 16:00:50.570746912 +0100
@@ -291,7 +291,7 @@
 
 In the lower part of the password screen a text is displayed.  This
 message is taken from the first file of the following that exists:
-$HOME/.xlocktext, $HOME/.plan, or $HOME/.signature.
+$HOME/.xlocktext or $HOME/.signature.
 
 On systems which support new BSD style authentication, the password
 may be prefixed by an authentication style followed by a colon
